apps/group/app/src/components/FakeShell/FakeShell.js

```diff
@@ -71,6 +68,10 @@ const TopBarTitle = styled.h1`
   background: url(${chevron}) no-repeat 100% 50%;
 `
 
+const TopBarEndButton = styled(Button)`
+  width: 150px;
+`
```

Comment:
See https://github.com/aragon/aragon-ui/pull/19; we can't use reverse selectors directly with `Button`s.